### ISSUE TITLE
Further hyperparameter tweaks

### ISSUE THREAD

**Issue by jtanwk** [jtanwk/cv-semantic-segmentation#7]
- Learning rate
- Learning rate decay
- Dropout
